src/explore-education-statistics-common/src/modules/charts/components/LineChartLabel.tsx

```diff
@@ -0,0 +1,79 @@
+import { LineChartDataLabelPosition } from '@common/modules/charts/types/chart';
+import { LegendInlinePosition } from '@common/modules/charts/types/legend';
+import formatPretty from '@common/utils/number/formatPretty';
+import { LabelProps } from 'recharts';
+import React from 'react';
+
+interface Props extends LabelProps {
+  colour: string;
+  dataLabelPosition?: LineChartDataLabelPosition;
+  decimalPlaces?: number;
+  index: number;
+  name: string;
+  legendLabelPosition: LegendInlinePosition;
```

Comment:
This should be optional. I'd also potentially go bit further and merge this with `dataLabelPosition` as a single `position` prop:

```ts
position?: LineChartDataLabelPosition | LegendInlinePosition;
```

Which could be then called like:

```tsx
position={showDataLabels ? dataLabelPosition : config.inlinePosition}
```